web-wallet/src/lib/components/Stake/Stake.svelte

```diff
@@ -65,6 +65,12 @@
 	/** @type {boolean} */
 	let isNextButtonDisabled = false;
 
+	/** @type {number} */
+	let gasPrice;
```

Comment:
Shouldn't `gasPrice` and `gasLimit` be initialized with the received `gasSettings`?

e.g.
```js
let { gasLimit, gasPrice } = gasSettings;
```